src/verilator/IMem.hh

```diff
@@ -0,0 +1,17 @@
+#ifndef __VERILATOR_IMEM__HH__
+#define __VERILATOR_IMEM__HH__
+
+#include "mem/mem_object.hh"
+
+//verilator file should be here
+
+class IMem
+{
+        protected:
```

Comment:
You should be using 4 spaces for tabs. See the style guideline (http://gem5.org/Coding_Style)